.github/workflows/macos_aarch64.yml

```diff
@@ -10,8 +10,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [macOS-latest]
-        arch: [x86_64, arm64]
+        os: [macos-14]
+        arch: [arm64]
```

Comment:
why?

Reply:
Because macos-latest means macos-12, which only has x86 support.

ref: https://github.com/actions/runner-images

Image | YAML Label | Included Software
-- | -- | --
Ubuntu 22.04 | ubuntu-latest or ubuntu-22.04 | ubuntu-22.04
Ubuntu 20.04 | ubuntu-20.04 | ubuntu-20.04
macOS 14 [beta] | macos-14-large | macOS-14
macOS 14 Arm64 [beta] | macos-14 or macos-14-xlarge | macOS-14-arm64
macOS 13 | macos-13, macos-13-large or macos-latest-large | macOS-13
macOS 13 Arm64 | macos-13-xlarge | macOS-13-arm64
macOS 12 | macos-latest,macos-12 ormacos-12-large | macOS-12
macOS 11 [deprecated] | macos-11 | macOS-11
Windows Server 2022 | windows-latest or windows-2022 | windows-2022
Windows Server 2019 | windows-2019 | windows-2019

Reply:
So I think we can use macos-12 for x86 macos, macos-14 for arm macos. (macos-13 x86 give weird error, skip for now)

Reply:
If we don't use the arm64 macos ci, the download jdk binary will give 'Bad CPU type in executable' error.

Reply:
ok